torch_frame/data/multi_nested_tensor.py

```diff
@@ -164,7 +164,8 @@ def _col_narrow(self, start: int, length: int) -> MultiNestedTensor:
         accum = torch.cumsum(offset_mat_zero_start[:, -1], dim=0)
         offset_mat_zero_start[1:] += accum[:-1].view(-1, 1)
         num_cols = end - start
-        offset = torch.full((self.num_rows * num_cols + 1, ), accum[-1])
+        offset = torch.full((self.num_rows * num_cols + 1, ),
+                            cast(int, accum[-1]))
```

Comment:
For reviewers, `typing.cast` is no-op at runtime and so introduces no impact on performance. See https://docs.python.org/3/library/typing.html#typing.cast as needed.